atmat/attests/pytests.m

```diff
@@ -90,7 +90,7 @@ function orbit6(testCase,lat2)
             porbit6=double(porbit6)';
             % Matlab
             [~,morbit6]=findorbit6(lattice.m);
-            testCase.verifyEqual(morbit6,porbit6,AbsTol=1.E-15 );
+            testCase.verifyEqual(morbit6,porbit6,AbsTol=5.E-15 );
```

Comment:
Did anything changed in the computation of the closed orbit? Do you get problems with the previous value?

Reply:
I did not change anything but the matlab test was systematically failing so I just matched the tolerance of the matlab test to the one of the python test.

Reply:
Failing on GitHub ? It still succeeds on the other branches, so it looks strange…

Reply:
So there is one thing. `energy_loss.py` was changed and is used to determine the initial condition for `find_orbit6`. This could have an impact on the final result to that level no?

Reply:
Ok, that could be an explanation. Do you confirm that it fails on GitHub (which platform?), because it still works for me (MacOS).

Reply:
I found the last failure in the GitHub history: it failed on ubuntu-latest, with an error of 3.2e-15 instead of 1e-15. We don't have the results on the other platforms because all runs are cancelled as soon as one fails.

So let's assume `energy_loss.py` is the reason for the change.

Reply:
The matlab tests fail on GitHub, I could not install the matlabengine in my pyenv, either it complains about the matlab version or the python version so I gave up.